Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11-slim
```

Comment:
Is there a reason to not use the latest Python (currently 3.13) here?

Reply:
I don't recall, but probably not, must be an oversight on my part. Updated.